src/templates/static.tsx

```diff
@@ -162,6 +239,7 @@ const StaticPage = ({
   data: { siteData, pageData: mdx },
   pageContext: { relativePath },
 }: PageProps<Queries.StaticPageQuery, Context>) => {
+  const theme = useTheme()
```

Comment:
Remove per [Change of this line](https://github.com/ethereum/ethereum-org-website/pull/9631/files#diff-408585bd7dc868312be9fcca1b88470e7da58536cadfc717c313fed65f18fe51R294)